sklearn/linear_model/logistic.py

```diff
@@ -1399,6 +1399,9 @@ class LogisticRegression(BaseEstimator, LinearClassifierMixin,
     cases. See :ref:`differences from liblinear <liblinear_differences>`
     in the narrative documentation.
 
+    To evaluate the significance of coefficients, one may use bootstrapping.
```

Comment:
but there's nothing in the documentation about that right now, right?

Reply:
No we are working on it now

Reply:
Well I thought we wanted to add a link to statsmodels first?